resources/tentative/complex-static-html/src/react-todomvc/todo/components/footer.jsx

```diff
@@ -0,0 +1,41 @@
+import { useCallback, useMemo } from "react";
+import { useLocation } from "react-router-dom";
+import classnames from "classnames";
+
+export function Footer({ todos, dispatch }) {
+    const { pathname: route } = useLocation();
+
+    const activeTodos = useMemo(() => todos.filter((todo) => !todo.completed), [todos]);
+
+    const removeCompleted = useCallback(() => dispatch({ type: "REMOVE_COMPLETED_ITEMS" }), [dispatch]);
+
+    // prettier-ignore
+    if (todos.length === 0)
```

Comment:
We typically don't compare against 0 like this. Use !todos.length instead:
https://webkit.org/code-style-guidelines/#zero-comparison

Reply:
Done.